src/tests/baseservices/compilerservices/UnsafeAccessors/UnsafeAccessorsTests.cs

```diff
@@ -188,6 +215,23 @@ public static void Verify_AccessFieldClass()
         extern static ref string GetPrivateField(UserDataClass d);
     }
 
+    [Fact]
+    [ActiveIssue("https://github.com/dotnet/runtime/issues/92633")]
```

Comment:
Do these tests fail on all runtimes or just on CoreCLR? If the tests fail on CoreCLR only, we can add `TestRuntimes.CoreCLR` filter to the ActiveIssue.

Reply:
They fail on all runtimes.